standalone-tools/wss-scan.sh

```diff
@@ -0,0 +1,126 @@
+###### Information ############################################################################
+# Name:          wss-scan.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to scan the odfe distros for vulnerabilities and licenses
+#                It will scan the repositories and send the WhiteSource link to the mail 
+#                of the user. 
+#
+# Prerequisites: Need to install Java 14
+#                Export JAVA_HOME env variable to the JDK path
+#		 Add JAVA_HOME to PATH variable
+#                Need to set the recepient mail in wss-scan.config for local run
+#                API token is needed for local run
+#
+# Usage:         ./wss-scan.sh
+#
+# Starting Date: 10-13-2020
+# Modified Date: 10-15-2020
+###############################################################################################
+
+set -e
+
+java -version
+if [ $? != 0 ] ; then echo "Java has not been setup" ; exit ; fi
+
+# download the WhiteSource Agent 
+curl -LJO -sS https://github.com/whitesource/unified-agent-distribution/releases/latest/download/wss-unified-agent.jar
+
+# The version 20.9.2.1 has been tested and can be used if a specific version is required
+#curl -LJO -sS https://github.com/whitesource/unified-agent-distribution/releases/download/v20.9.2.1/wss-unified-agent.jar
+
+
+echo "wss-unified-agent jar download has been completed"
```

Comment:
curl does not return failures better actually check if it is complete or use wget instead.